docs/walkthroughs/setting-an-external-dmz-url.adoc

```diff
@@ -0,0 +1,36 @@
+[[setting-an-external-dmz-url]]
+= Setting an external DMZ URL
+
+== Overview
+
+Often in an enterprise setup, our infrastructure and services sit on internal networks that can not be accessed directly from the outside world. This means that the mobile client we create can only access services we provision by being on the internal network or by using some form of proxy. Most often these proxies live in something known as the DMZ or demilitarized zone. Genrally the DMZ is where all incoming traffic is handled and our services exposed. When talking to a proxy the URLs you use to configure your mobile client applicaiton, will be different than those of the services on the internal network. Setting the DMZ url on your mobile client within the OpenShift UI or via the mobile CLI will allow you to get an updated mobile client configuration that uses the external DMZ url instead of the internal service routes.
+
```

Comment:
applicaiton --> application typo. Description is fine and I think it is explained simply but clearly